src/barkhaPJun20/FindMissingNumInArray.java

```diff
@@ -0,0 +1,35 @@
+package barkhaPJun20;
+
+public class FindMissingNumInArray {
+	int arraySum=0;
+	int originalSum=0;
+	//Method 1:Find Missing number by find difference in addition.
+	void printMissingNumInArray(int[] arrayNum) {
+		for (int index=0; index<arrayNum.length; index++) {
+			arraySum=arraySum+arrayNum[index];
+		}
+		for (int num=1; num<=10; num++) {
+			originalSum=originalSum+num;
```

Comment:
Here you can write n(n+1)/2 formula as well.
void findMissingNumber2(int[] array) {
		int al = array.length+1;
		int expectedSum = (al*(al+1))/2;
		int actualSum = 0;
		for(int index=0; index<al-1; index++) {
			actualSum = actualSum + array[index];
		}
		System.out.println("expectedSum "+expectedSum);
		System.out.println("Missing number in given array is: "+(expectedSum-actualSum));
	}